src/main/java/menelaus/view/MainScreen.java

```diff
@@ -0,0 +1,103 @@
+package menelaus.view;
+
+import java.awt.BorderLayout;
+import java.awt.EventQueue;
+
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.border.EmptyBorder;
+import javax.swing.JTextPane;
+import javax.swing.JButton;
+import javax.swing.JTextField;
+import java.awt.event.ActionListener;
+import java.awt.event.ActionEvent;
+import javax.swing.GroupLayout;
+import javax.swing.GroupLayout.Alignment;
+import javax.swing.LayoutStyle.ComponentPlacement;
+
+public class MainScreen extends JFrame {
+
+	private JPanel contentPane;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'contentPane' could be replaced by a local variable.](https://www.codacy.com/app/dmvouldjeff/menelaus/file/2401080830/issues/source?bid=3228196&fileBranchId=3232364#l20)